src/main/java/tuna/command/AddEventCommand.java

```diff
@@ -0,0 +1,50 @@
+package tuna.command;
+
+import java.util.Arrays;
+
+import tuna.Storage;
+import tuna.TaskList;
+import tuna.TunaException;
+import tuna.Ui;
+
+/**
+ * Represents an add Event Task command. An AddEventCommand object contains the index of the /at command,
+ * task description and when the event is occurring.
+ */
+public class AddEventCommand extends Command {
+    /** Index of the /at command */
+    private int limit;
+    /** Description of the task */
+    private String taskDescription;
+    /** When the event is occurring */
+    private String at;
```

Comment:
Being a bit picky here, but perhaps can change the variable name from 'at' to' 'time'? At is a bit vague.